CorsixTH/Src/iso_fs.cpp

```diff
@@ -371,6 +371,9 @@ void iso_filesystem::set_path_separator(char cSeparator) {
 
 bool iso_filesystem::initialise(const char* path) {
   clear();
+// Suppress invalid MSVC warning
+// https://github.com/CorsixTH/CorsixTH/issues/1961#issuecomment-868573582
+#pragma warning(suppress : 4996)
```

Comment:
This applies to the whole file below this point. You may want to move it to the top of the file or undo the suppress after the line. If moving to the top of the file, link to the warning please.

Reply:
Seems like the warning is a general security one so best suppressed just for this line. Also we shouldn't use this pragma outside of MSVC.